traceview/__init__.py

```diff
@@ -221,8 +223,42 @@ def delete(self, host_id, *args, **kwargs):
           True
 
         """
+        return self.delete_host(host_id, *args, **kwargs)
+
+    def delete_host(self, host_id, *args, **kwargs):
+        """ Delete an existing host.
+
+        :param str host_id: The id of the host to delete.
```

Comment:
I thought the same thing--no reason for it to be a string, should be an int.  I kept the way it is for simplicity but I'll just sneak it in.